rxgroups/src/main/java/com/airbnb/rxgroups/ObservableManager.java

```diff
@@ -27,12 +28,17 @@
  * Subscribe to observables, and then lock or unlock their observers to control when you get the
  * event back. Events will be held in a queue until an Observer is added and the group is unlocked.
  */
+@SuppressWarnings("WeakerAccess")
 public class ObservableManager {
   /** Map ids to a group of observables. */
   private final Map<Long, ObservableGroup> observableGroupMap = new ConcurrentHashMap<>();
   private final AtomicLong nextId = new AtomicLong(1);
+  private final UUID uuid = UUID.randomUUID();
 
-  /** @return an existing group or a new group with the provided groupId */
+  /**
+   * @return an existing group or a new group with the provided groupId. Throws {@link
```

Comment:
`existing group or a new group with the provided groupId` does that make sense if it throws if no group with that id exists? it makes it sound like it creates a group with the id if it doesn't exist